src/components/FloatingActionButton/FloatingActionButton.js

```diff
@@ -0,0 +1,30 @@
+import "./FloatingActionButton.css";
+import { useState, useEffect } from "react";
+import add from "../../assets/add.svg";
+
+const FloatingActionButton = ({ onClick }) => {
+  const [showFab, setShowFab] = useState(false);
+
+  useEffect(() => {
+    const handleResize = () => {
+      const screenWidth = window.innerWidth;
+      setShowFab(screenWidth <= 768);
+    };
+    handleResize();
+    window.addEventListener("resize", handleResize);
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
+
+  return showFab ? (
+    <>
+      <button className="fab" onClick={onClick}>
+        폴더 추가
+      </button>
+      <img src={add} className="floating-add-button" alt="add" />
+    </>
+  ) : null;
```

Comment:
```suggestion
 if (!showFab) {
   return null;
 }

  return  (
    <>
      <button className="fab" onClick={onClick}>
        폴더 추가
      </button>
      <img src={add} className="floating-add-button" alt="add" />
    </>
  )
```
[중요도: 높음] 만약 특정 조건이 만족하지 못할 때 아무것도 노출시켜주지 않게 하려면 early return 패턴을 이용하시는 것을 추천드립니다. 조금 더 가독성이 좋아집니다.

[중요도: 높음] return null로 아무것도 없는 컴포넌트를 처리하는 이유가 뭘까요? 다른 방식은 없을까요?